connectors/spark/src/test/java/io/unitycatalog/spark/auth/AbfsVendedTokenProviderTest.java

```diff
@@ -19,22 +18,14 @@
 public class AbfsVendedTokenProviderTest extends BaseTokenProviderTest<AbfsVendedTokenProvider> {
   @Override
   protected AbfsVendedTokenProvider createTestProvider(
-      Clock clock,
-      long renewalLeadTimeMillis,
```

Comment:
we just changed to set configuration keys in the hadoop configuration (for both `clock` and `renewLeadTimeMillis` ). So we don't need those contructors any more. 

As I saide above here: https://github.com/unitycatalog/unitycatalog/pull/1123/files#r2461390293